docs/Makefile

```diff
@@ -20,3 +20,16 @@ html: Makefile
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+# Gen man pages
+ZFS_GIT_REPO = https://github.com/openzfs/zfs.git
+ZFS_GIT_DIR = ./_build/zfs
+
+.PHONY: man
+man:
+	if [ ! -d $(ZFS_GIT_DIR) ]; \
+		then git clone --depth=1 $(ZFS_GIT_REPO) ./_build/zfs; \
```

Comment:
nit: can you move this `then` to the previous line, after the `;`, as is traditional.